src/util/math.hpp

```diff
@@ -63,4 +65,24 @@ namespace big::math
 		SHAPETEST::GET_SHAPE_TEST_RESULT(ray, &hit, &end_coords, &surface_normal, &hit_entity);
 		return end_coords;
 	}
+
+	// [0, max_value)
+	inline int rand(int max_value)
+	{
+		std::random_device seed;
+		std::mt19937 gen{seed()};
```

Comment:
Seeding every time we call rand? Sounds like a bad idea.

Probably best to make `seed` and `gen` static variables, correct me if I'm wrong though.

Reply:
Could probably use a singleton pattern to initalize the seed.

Reply:
Yeah using static isn't really the best way, but better than the current.